.pylintrc

```diff
@@ -60,7 +60,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=
+disable=consider-using-f-string
```

Comment:
This should get us away from this back and forth issue linter issues when we don't put an f-string in the call to the log.